alpha-build-machinery/Makefile

```diff
@@ -11,7 +11,9 @@ examples :=$(wildcard ./make/examples/*/Makefile.test)
 # We need to change dir to the final makefile directory or relative paths won't match
 define update-makefile-log
 mkdir -p "$(3)"
-$(MAKE) -C "$(dir $(1))" -f "$(notdir $(1))" --no-print-directory --warn-undefined-variables $(2) 2>&1 | tee "$(3)"/"$(notdir $(1))"$(subst ..,.,.$(2).log)
+$(MAKE) -j 1 -C "$(dir $(1))" -f "$(notdir $(1))" --no-print-directory --warn-undefined-variables $(2) 2>&1 | \
```

Comment:
what does this do? needs a comment

Reply:
`-j 1` means serial (jobs) - so it is deterministic in tests and the diffs match

Reply:
I meant the whole 3 line monster, especialy the sed

Reply:
the sed replaces stuff like the build date (which differs every invocation)  with a string "<redacted_for_diff>" or the checkout templates would never match